scripts/webpack/browser.js

```diff
@@ -127,3 +116,14 @@ module.exports = ({ isLegacyJS }) => ({
 		],
 	},
 });
+
+module.exports.babelExclude = {
+	and: [/node_modules/],
+	not: [
+		// Include all @guardian modules, except automat-modules
+		/@guardian\/(?!(automat-modules))/,
+
+		// Include the dynamic-import-polyfill
+		/dynamic-import-polyfill/,
+	],
+};
```

Comment:
I had to move this to the bottom of the file, as `module.exports = ...` was overwriting the property and you couldn't import `babelExclude` in other files.